conda/testing/fixtures.py

```diff
@@ -306,6 +310,110 @@ def session_conda_cli() -> Iterator[CondaCLIFixture]:
     yield CondaCLIFixture(None)
 
 
+@dataclass
+class PipCLIFixture:
+    """Fixture for calling pip in specific conda environments."""
+
+    capsys: CaptureFixture | None
```

Comment:
do we actually need `capsys` since this is a `subprocess.run(capture_output=True)`?